src/app/components/ThemeProvider/themes/afaanoromoo.ts

```diff
@@ -0,0 +1,14 @@
+import { GHOST, WHITE, POSTBOX, POSTBOX_30 } from '../colours';
+import withThemeProvider from '../withThemeProvider';
+
+const afaanoromooTheme = {
+  colours: {
```

Comment:
Wonder if it's worth considering the american spelling here, for conformity with CSS and other libraries

Reply:
I was thinking about this. I was contemplating. using `palette` to avoid this issue.

Reply:
make it so